lib/srv/reexec_other.go

```diff
@@ -24,3 +24,5 @@ import (
 func reexecCommandOSTweaks(cmd *exec.Cmd) {}
 
 func userCommandOSTweaks(cmd *exec.Cmd) {}
+
+func adjustOOMScore() error {}
```

Comment:
```suggestion
func adjustOOMScore() error { return nil }
```

Reply:
Good point. The code should still compile 😅